zerver/openapi/zulip.yaml

```
@@ -16976,6 +16988,38 @@ components:
                   The unique message ID. Messages should always be
                   displayed sorted by ID.
     BasicStream:
+      allOf:
+        - $ref: "#/components/schemas/BasicStreamBase"
+        - additionalProperties: false
+          properties:
+            stream_id: {}
+            name: {}
+            description: {}
+            date_created: {}
+            invite_only: {}
+            rendered_description: {}
+            is_web_public: {}
+            stream_post_policy: {}
+            message_retention_days:
+              nullable: true
+            history_public_to_subscribers: {}
+            first_message_id:
+              nullable: true
+            is_announcement_only: {}
+            can_remove_subscribers_group: {}
+            stream_weekly_traffic:
+              type: integer
+              nullable: true
+              description: |
+                The average number of messages sent to the stream in recent weeks,
+                rounded to the nearest integer.
```

```suggestion
                The average number of messages sent to the stream per week,
                as estimated based on recent weeks, rounded to the nearest integer.
```

(The "per week" is important for understanding exactly how to interpret the number. The "recent weeks" part is also useful but is less critical.)